docs/install.md

```diff
@@ -5,11 +5,41 @@
 
 * **Python**: Required. Model Server for Apache MXNet (MMS) works with Python 2 or 3.  When installing MMS, we recommend that you use a Python and Conda environment to avoid conflicts with your other Apache MXNet or Open Neural Network Exchange (ONNX) installations.
 
-* **protoc**: Optional. If you plan to use ONNX features, you need to install the [protobuf compiler](https://github.com/onnx/onnx#installation). Install it *before* installing MMS.
+* **java 8**: Required. MMS use java to serve HTTP requests. You must install java 8 (or later) and make sure java is on available in $PATH environment variable *before* installing MMS. If you have multiple java installed, you can use $JAVA_HOME environment vairable to control which java to use.
+
+For ubuntu:
+```bash
+sudo apt-get install openjdk-8-jre-headless
+```
+
+For centos
+```bash
+sudo yum install java-1.8.0-openjdk
+```
+
+For Mac:
+```bash
+brew tap caskroom/versions
+brew update
+brew cask install java8
+```
+
+* **MxNet**: Recommended. MMS won't install `mxnet` by default. MxNet is required for most of examples in this project. MMS won't install mxnet engine by default, you can install mxnet-mkl or mxnet-cu90mkl based on your need. And you can also choose specific version of mxnet if you want.
+
+```bash
+pip install mxnet-mkl
+```
+
+or for GPU instance:
+
+```bash
+pip install mxnet-cu90-mkl
+```
+
 
 * **Curl**: Optional. Curl is used in all of the examples. Install it with your preferred package manager.
 
-* **Unzip**: Optional. Unzip allows you to easily extract model files and inspect their content. If you choose to use it, associate it with `.model` extensions.
+* **Unzip**: Optional. Unzip allows you to easily extract model files and inspect their content. If you choose to use it, associate it with `.mar` extensions.
 
 ## Install MMS with pip
 
```

Comment:
add mxnet installation option here as well?
eg: pip install mxnet-model-server[mxnet-mkl]

Reply:
I didn't find a good way to add this. Will do another round of scrub with Aaron.